Makefile

```diff
@@ -3,7 +3,9 @@
 MOCHA_OPTS= --check-leaks
 REPORTER = dot
 
-test: test-unit test-integration
```

Comment:
Ah- when they're up here, they're calling the other targets as dependencies of this one.